src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -1968,6 +1968,20 @@ def test_check_failure_with_comparison(self):
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
 
+  def test_check_failure_with_comparison2(self):
+    """Test for special CHECK failure formats (CHECK_EQ, CHECK_LE, etc.)."""
+    data = self._read_test_data('check_failure_with_comparison2.txt')
+    expected_type = 'CHECK failure'
+    expected_address = ''
+    expected_state = ('layout_snapped_paint_offset == snapped_paint_offset '
+                      'in compositing_layer_propert\n')
```

Comment:
Note that this truncation is due to our 80 char line limit. i can think of something better later but need to think of way to not break existing test, so that fix can come later.

Reply:
Thanks for explaining this!